docs/translations.rst

```diff
@@ -7,3 +7,12 @@ to contribute.
 
 If you are experiencing issues as a translator, please let us know by opening a
 `translation issue on the Warehouse issue tracker <https://github.com/pypa/warehouse/issues/new?template=translation-issue.md>`_.
+
+To add a new translation:
+
+- Merge the pull request from Weblate
```

Comment:
How are Weblate PRs going to be merged? Personally, I’d suggest squashed, because I tend to accidentally spam Git logs by my sparse edits of just a few strings.

Reply:
To answer my own question: yup, squashed, deriving from the precedent of how #6667 was merged. =)